lib/dry/inflector.rb

```diff
@@ -34,20 +34,50 @@ def initialize(&blk)
       @inflections = Inflections.build(&blk)
     end
 
-    # Camelize a string
+    # Lower camelize a string
     #
     # @param input [String,Symbol] the input
-    # @return [String] the camelized string
+    # @return [String] the lower camelized string
     #
-    # @since 0.1.0
+    # @since 0.1.2
+    #
+    # @example
+    #   require "dry/inflector"
+    #
+    #   inflector = Dry::Inflector.new
+    #   inflector.camelize_lower("dry/inflector") # => "Dry::Inflector"
+    def camelize_lower(input)
+      input.to_s.
+        gsub(/\/(.?)/) { "::#{Regexp.last_match(1).upcase}" }.
```

Comment:
What about to move this common part to the private method, like `#start_camelizing`?

Reply:
:+1:

I'll go back to computer later this morning. Im at GMT -2:00

Reply:
:+1:

I'll go back to computer later this morning. Im at ~GMT -2:00~ GMT -3:00.

Reply:
Better. Feel free to ammend it if you want. :+1:

Reply:
Something like this…

Reply:
👍